mkdocs.yml

```diff
@@ -7,7 +9,7 @@ theme:
   hljs_languages:
     - yaml
 nav:
-  - 'readme.md'
+  - 'Home': user.md
```

Comment:
> When I test with `mkdocs serve` and I go to the base domain (just localhost:8000 for me) it shows a 404. I think this will happen on the live site as well. I think we need an `index.md`. What do you think?

yeah, I just [commented](https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/24#discussion_r1013011007), and added the README.md on the last commit.